src/core/plugins/oas3/components/request-body.jsx

```diff
@@ -132,6 +124,18 @@ const RequestBody = ({
     return <Input type={"file"} onChange={handleFile} />
   }
 
+
+  if (!mediaTypeValue.size) {
+    return null
+  }
+
+  const schemaType = mediaTypeValue.getIn(["schema", "type"])
+  const isObjectContent =
```

Comment:
as discussed we'll abstract this later with something like `includesType(schema, [type1, type2])`